### ISSUE TITLE
Wrong music in No Rest for the Living

### ISSUE THREAD

**Issue by SoddenCub71** [fabiangreffrath/crispy-doom#958]
### Background

Version of Crispy Doom: 5.12.0

Operating System and version: Windows 11 Home 22H2

Game: Doom II

Any loaded WADs and mods (please include full command line):
Rather than launching by command line, I drag the file I've named "NRftL.wad" onto crispy-doom.exe. The file is the version of No Rest for the Living available through the "Doom Classic Unity" version of Doom II.
[NRftL.zip](https://github.com/fabiangreffrath/crispy-doom/files/9712967/NRftL.zip)

I also have the music pack and extras.wad from that version in place.

### Bug description

Observed behavior:
Levels play the music tracks of their correspondingly numbered Doom II levels; e.g. MAP01: The Earth Base plays "Running From Evil"/D_RUNNIN.ogg.
https://user-images.githubusercontent.com/114965480/193988513-15fe6113-395a-42f5-bcf6-43ad59846447.mp4

Expected behavior:
Levels play the same music tracks that they do when played under the official engine; as listed in the table at https://doomwiki.org/wiki/No_Rest_for_the_Living#Levels.
https://user-images.githubusercontent.com/114965480/193988348-6a797100-ced6-41c3-93d2-990b355e478b.mp4

**Comment by mikeday0** [comment]
Try renaming the wad file `nerve.wad`.

**Comment by SoddenCub71** [comment]
> Try renaming the wad file `nerve.wad`.

Never would have thought to try that, but it worked.

Do other wads play the wrong music if they have the wrong filename? Is that something that can/should be fixed? Where can I get the "canonical" filenames for the "official" add-ons, since the game just downloads them as "1", "2" and so on? Are they always prefixed to attached screenshots, so I just need to cut the last digit from the screenshots' filenames?
![image](https://user-images.githubusercontent.com/114965480/194146084-19c3f093-9e20-4582-9583-45eac84366eb.png)

**Comment by fabiangreffrath** [comment]
This will be helpful:
https://doomwiki.org/wiki/Category:Resources

**Comment by SoddenCub71** [comment]
> This will be helpful: https://doomwiki.org/wiki/Category:Resources

That's a lot of pages. Do I need to read all of them, just to play add-ons the way they were intended?

**Comment by SoddenCub71** [comment]
> The .WAD files enumerated in that list are all the canonical names for the official wads.

I only see 4 of the 16 official add-ons there. Does that mean that the others don't have canonical names, and can be renamed at will?

**Comment by mikeday0** [comment]
Yes, I think it should be fine to rename those. It’s really just `nerve.wad` and `masterlevels.wad` which have special code paths that are triggered by file name.

**Comment by SoddenCub71** [comment]
> Yes, I think it should be fine to rename those. It’s really just `nerve.wad` and `masterlevels.wad` which have special code paths that are triggered by file name.

Good to know.

I'll still keep the issue open in the hope that someone's willing to patch the behavior someday; assuming there isn't some technical or design reason that they can't.

**Comment by SoddenCub71** [comment]
> Yes, I think it should be fine to rename those. It’s really just `nerve.wad` and `masterlevels.wad` which have special code paths that are triggered by file name.

One other thing that depends on filename is level names (as shown on the map).
![image](https://user-images.githubusercontent.com/114965480/194187175-a7ef2f60-339b-42f5-a542-99e41d896c25.png)
![image](https://user-images.githubusercontent.com/114965480/194187125-ce51db8b-df8e-4b5f-bd24-8f529317169b.png)
